common.py

```diff
@@ -5,16 +5,16 @@
 import csv
 from datetime import datetime
 import re
-from typing import Iterable, Union
+from typing import Iterable, Union, Dict
 
 
 # Commonly used dimensions
 COURTYARD_LINE_WIDTH = 0.1
 
 
-def init_cache(uuid_cache_file: str) -> collections.OrderedDict:
+def init_cache(uuid_cache_file: str) -> Dict[str, str]:
     print('Loading cache: {}'.format(uuid_cache_file))
-    uuid_cache = collections.OrderedDict()
+    uuid_cache = collections.OrderedDict()  # type: Dict[str, str]
```

Comment:
The comment annotation is also only needed for Python 3.5 support. Python 3.7 would support `uuid_cache: Dict[str, str]`.